Gemfile

```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
-ruby '2.2.2'
+ruby '2.4.7'
 gem 'puma'
-gem 'rails', '4.2.3'
-gem 'pg'
+gem 'rails', '4.2.10'
+gem 'pg', '~> 0.18'
```

Comment:
Without the `'~> 0.18'` there were errors to add the pg gem, and ensure it was at a version supported by ActiveRecord. Found this by googling and it fixed it.